packages/ss58_codec/test/ss58_codec_test.dart

```diff
@@ -56,78 +56,178 @@ void main() {
     }
   });
 
-  // Decode Exception testing
+  /// [SS58Codec] `decode` method Exception testing
   group('decode: throw exception -> ', () {
+    /// `String` address with `length < 3`
     test('data.length < 3 ', () {
```

Comment:
I strongly recommend[ BDD Unit test naming conventions ](https://matheus.ro/2017/09/24/unit-test-naming-convention/) when naming tests. Tests should be readable and have a well defined behavior. This applies to all unit tests in this PR lol
```suggestion
    test('Should throw exception when data lenght is too small', () {
```

Reply:
Highly agree with this. We should be able to know what is the goal of the tests only by their name. Even if the name is a bit big.